### PR TITLE
Update documentation for MKDocs

### PR DESCRIPTION
**Pull request by RickVanRousselt**
Linked issue [#238]
This will be a V1 because certain things will have to be updated (tutorials, user guides, analytics,...)  afterwards.
But let's first setup the deployment of the docs

### CODE REVIEW

File: docs/docs/contributing/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+* [Embedded JavaScript templating](http://ejs.co) - EJS is the core engine used in Yeoman to deploy template files and dynamically embed custom settings in the files getting deployed
+
+## Places to look for help
+The JavaScript a Yeoman generator uses is based on Node.js and many things are treated differently than in the JavaScript that exists in browsers.
+
+* [Yeoman API Documentation](http://yeoman.io/generator/) - 
+Besides the getting started guide, there is official documentation available that covers the complete Yeoman API
+
+* [Node.js documentation](https://nodejs.org/dist/latest-v8.x/docs/api/) -
```
Comment (wictorwilen) [inline docs/docs/contributing/getting-started.md]: Should link to node LTS and not 8.x (since that is not supported anymore)

**Review by wictorwilen — CHANGES_REQUESTED** [review]
Awesome work!

File: docs/docs/contributing/getting-started.md (hunk 1 above)
Comment (RickVanRousselt) [reply docs/docs/contributing/getting-started.md]: Changed to the general Node docs page. That way we don't need to update this page every time the LTS version changes ;-)

**Review by wictorwilen — APPROVED** [review]
Good to go!

**Pull request merged**